config/pmd-main.xml

```diff
@@ -54,6 +54,8 @@
                       |//ClassOrInterfaceDeclaration[@Image='CheckstyleAntTask']
                                  //MethodDeclaration[@Name='getListeners'
                                                        or @Name='createDefaultLogger']
+                      |//ClassOrInterfaceDeclaration[@Image='XdocsUrlTest']
```

Comment:
I try to add this to suppress CloseResource violation in pmd, but it not effects. I need some help